x/marketmaker/client/cli/query.go

```diff
@@ -0,0 +1,180 @@
+package cli
+
+// DONTCOVER
+// client is excluded from test coverage in MVP version
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+
+	"github.com/cosmosquad-labs/squad/v2/x/marketmaker/types"
+)
+
+// GetQueryCmd returns a root CLI command handler for all x/marketmaker query commands.
+func GetQueryCmd() *cobra.Command {
+	farmingQueryCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "Querying commands for the marketmaker module",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	farmingQueryCmd.AddCommand(
+		GetCmdQueryParams(),
+		GetQueryMarketMakerCmd(),
+		GetCmdQueryIncentive(),
+	)
+	return farmingQueryCmd
+}
+
+// GetCmdQueryParams implements the query params command.
+func GetCmdQueryParams() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "params",
+		Args:  cobra.NoArgs,
+		Short: "Query the current market maker parameters information",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Query values set as market maker parameters.
+Example:
+$ %s query %s params
+`,
+				version.AppName, types.ModuleName,
+			),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			queryClient := types.NewQueryClient(clientCtx)
+
+			resp, err := queryClient.Params(cmd.Context(), &types.QueryParamsRequest{})
+			if err != nil {
+				return err
+			}
+
+			return clientCtx.PrintProto(&resp.Params)
+		},
+	}
+
+	flags.AddQueryFlagsToCmd(cmd)
+
+	return cmd
+}
+
+// GetQueryMarketMakerCmd implements the market maker query command.
+func GetQueryMarketMakerCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "marketmaker",
```

Comment:
Suggestion: `"marketmakers"` since the original query method is named `MarketMakers` and it returns a list.